experimental/dds/tree2/src/test/snapshots/files/empty-root-final.json

```diff
@@ -14,7 +14,101 @@
               "String": {
                 "type": "blob",
                 "content": {
-                  "trunk": [],
+                  "trunk": [
+                    {
+                      "change": {
+                        "changes": [
+                          {
+                            "schema": {
+                              "new": {
+                                "version": "1.0.0",
+                                "nodeSchema": [
+                                  {
+                                    "name": "com.fluidframework.leaf.boolean",
+                                    "objectNodeFields": [],
+                                    "leafValue": 2
+                                  },
+                                  {
+                                    "name": "com.fluidframework.leaf.handle",
+                                    "objectNodeFields": [],
+                                    "leafValue": 3
+                                  },
+                                  {
+                                    "name": "com.fluidframework.leaf.null",
+                                    "objectNodeFields": [],
+                                    "leafValue": 4
+                                  },
+                                  {
+                                    "name": "com.fluidframework.leaf.number",
+                                    "objectNodeFields": [],
+                                    "leafValue": 0
+                                  },
+                                  {
+                                    "name": "com.fluidframework.leaf.string",
+                                    "objectNodeFields": [],
+                                    "leafValue": 1
+                                  },
+                                  {
+                                    "name": "test trees.TestInner",
+                                    "mapFields": {
+                                      "kind": "Sequence"
+                                    },
+                                    "objectNodeFields": []
+                                  }
+                                ],
+                                "rootFieldSchema": {
+                                  "kind": "Sequence"
+                                }
+                              },
+                              "old": {
```

Comment:
the new and old schema here look the same. That seems wrong to me. Shouldn't old be the default empty document schema? Either that of this change shouldn't have been done in the first place.

Reply:
Fixed! Nice catch. As discussed, I will add a test for this issue.

Reply:
Done.